src/main/java/wheresmymoney/ExpenseList.java

```diff
@@ -64,4 +64,20 @@ public void deleteExpense(int index) throws WheresMyMoneyException {
         }
         expenses.remove(index);
     }
+
+    /**
+     * Returns the list of all expenses from the specified category
+     *
+     * @param category Category of expense
+     */
+    public static Expense[] listByCategory(String category) {
```

Comment:
yeah also try not to make static methods for expenseList since its supposed to be an object